lib/feeder_importer.rb

```diff
@@ -0,0 +1,72 @@
+# import script for a podcast live in feeder & dovetail
+require 'prx_access'
+
+class FeederImporter
+  include PRXAccess
+
+  attr_accessor :cms_account_path, :feeder_podcast_path
+
+  def initialize(account, podcast)
+    self.cms_account_path = account
+    self.feeder_podcast_path = podcast
+  end
+
+  def import
+    # Query the feeder podcast, get info needed to create a series
+    podcast = get_feeder_podcast
+
+    # Create the series
+    series = create_series(podcast)
+
+    # Create the distribution
+    create_distribution(podcast, series)
+
+    # Create the episodes
+    response = podcast.episodes.get
+    create_episodes(response['_embedded']['prx:items'])
+    while response.links['next']
+      response = podcast.episodes.get
+      create_episodes(response['_embedded']['prx:items'])
+    end
+  end
+
+  def create_stories(episodes)
+    episodes.each { |e| create_story(e) }
+  end
+
+  def create_story(episode)
+    # create the story
```

Comment:
you can ignore this, not finished - I started working on it, but then changed to the podcast importer b/c we need it first.